src/portal/src/app/base/left-side-nav/replication/replication/list-replication-rule/list-replication-rule.component.html

```diff
@@ -136,6 +131,11 @@
                 {{ 'REPLICATION.DESCRIPTION' | translate }}
             </ng-template>
         </clr-dg-column>
+        <clr-dg-column>
+            <ng-template [clrDgHideableColumn]="{ hidden: hiddenArray[6] }">
```

Comment:
Could you update the order index of hiddenArray: 
current hiddenArray[7] to hiddenArray[6]
current hiddenArray[8] to hiddenArray[7]
current hiddenArray[6] to hiddenArray[8]

Then back to the list-replication-rule.component.ts : line:101
Update the following hiddenArray
    hiddenArray: boolean[] = getHiddenArrayFromLocalStorage(
        PageSizeMapKeys.LIST_REPLICATION_RULE_COMPONENT,
        [false, false, false, false, false, false, false, true, true]
    );
    
    to 
        hiddenArray: boolean[] = getHiddenArrayFromLocalStorage(
        PageSizeMapKeys.LIST_REPLICATION_RULE_COMPONENT,
        [false, false, false, false, false, false, true, true, false]
    );

Reply:
Sure, Fixed ✅